quna/src/main/java/com/ktpt/quna/application/exception/InvalidTokenException.java

```diff
@@ -0,0 +1,7 @@
+package com.ktpt.quna.application.exception;
+
+public class InvalidTokenException extends IllegalArgumentException {
+	public InvalidTokenException(String s) {
+		super(s);
+	}
+}
```

Comment:
```suggestion
}

```
맨 끝에 공백 라인이 하나 있어야 할 것 같네요!

Reply:
좋은 지적이에요!
총 3군데에서 발견했습니다.
커밋은 안하고 수정만 해놓겠습니다😬